tests/manage/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py

```diff
@@ -62,6 +62,56 @@ def is_node_rack_or_zone_exist(self, failure_domain, node_obj):
         """
         return get_node_rack_or_zone(failure_domain, node_obj) is not None
 
+    def check_rook_ceph_crashcollector_pods_where_rook_ceph_pods_are_running(self):
```

Comment:
I suggest we take this function out to a utils/helpers file as it might be useful for other stuff and not only for the test here

Reply:
Done